packages/p2p-v2/src/components/Modals/MyAdsDeleteModal/MyAdsDeleteModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { memo, useEffect } from 'react';
+import clsx from 'clsx';
+import Modal from 'react-modal';
+import { useDevice } from '@/hooks';
+import { p2p } from '@deriv/api';
+import { Button, Text } from '@deriv-com/ui';
+import { customStyles } from '../helpers';
+import './MyAdsDeleteModal.scss';
+
+type TMyAdsDeleteModalProps = {
+    error?: string;
+    id: string;
+    isModalOpen: boolean;
+    onClickDelete: () => void;
+    onRequestClose: () => void;
+};
+
+const MyAdsDeleteModal = ({ error, id, isModalOpen, onClickDelete, onRequestClose }: TMyAdsDeleteModalProps) => {
+    const { data: advertInfo, isLoading: isLoadingInfo } = p2p.advert.useGet({ id });
+    const { isMobile } = useDevice();
+
+    const hasActiveOrders = advertInfo?.active_orders && advertInfo?.active_orders > 0;
+
+    useEffect(() => {
+        Modal.setAppElement('#v2_modal_root');
+    }, []);
```

Comment:
That's a bit odd to me, can you explain how do we handle modals?
How about using `useModal`?

Reply:
as of now, we are using react-modal. after we have the modal component from deriv-com/ui this can be moved out or removed since we will be setting the app element from the root or index.